packages/react-composites/src/composites/MeetingComposite/state/MeetingParticipants.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { RemoteParticipantState } from '@internal/calling-stateful-client';
+import { CommunicationIdentifier } from '@azure/communication-common';
+import { MeetingEndReason } from './MeetingEndReason';
+
+export interface MeetingParticipant
+  extends Pick<RemoteParticipantState, 'displayName' | 'state' | 'videoStreams' | 'isMuted' | 'isSpeaking'> {
+  id: CommunicationIdentifier;
+  meetingEndReason?: MeetingEndReason;
+}
+
+export function createMeetingParticipant(callParticipant: RemoteParticipantState): MeetingParticipant {
```

Comment:
Naming nit: I prefer to use `newX` or `createX` for creating new objects that hold interanal state etc (i.e., factory functions).
This is simply data marshalling. Perhaps simply `meetingParticipantFromCallParticipant`?

Reply:
Updated to `meetingParticipantFromCallParticipant`